### PR TITLE
Added rds Snapshots table in aws

### PR DESCRIPTION
**Pull request by dheeraj-uptycs**
By submitting a PR to this repository, you agree to the terms within the [Code of Conduct](https://github.com/Uptycs/cloudquery/blob/master/CODE_OF_CONDUCT.md). Please see the [contributing guidelines](https://github.com/Uptycs/cloudquery/blob/master/CONTRIBUTING.md) for how to create and submit a PR for this repo.

### Description

Describe the purpose of this PR along with any background information and the impacts of the proposed change.

### References

Include any links supporting this change such as:

- GitHub Issue/PR number addressed or fixed
- Related pull requests/issues from other repos

### Testing

Describe the testing done. Setup required for testing. Details of unit or other tests added as a part of the PR.

### CODE REVIEW

**Comment by dheeraj-uptycs** [comment]
Description:
Added the aws_rds_snapshot table with all first level attributes to osquery. Columns have been added as per their target datatype , Added tables.md for rds folder in aws

References:
https://awscli.amazonaws.com/v2/documentation/api/latest/reference/rds/describe-db-snapshot-attributes.html

Testing:
Tested using osquery ,all the enabled attributes are fetched and Shown correctly with their respective datatype.

File: extension/aws/rds/aws_rds_snapshots.go [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+	paginator := rds.NewDescribeDBClusterSnapshotsPaginator(svc, params)
+
+	for {
+		page, err := paginator.NextPage(osqCtx)
+		if err != nil {
+			utilities.GetLogger().WithFields(log.Fields{
+				"tableName": "aws_rds_snapshots",
+				"account":   accountId,
+				"region":    *region.RegionName,
+				"task":      "DescribeDBSnapshots",
```
Comment (alende-uptycs) [inline extension/aws/rds/aws_rds_snapshots.go:161]: DescribeDBClusterSnapshots

File: extension/aws/rds/aws_rds_snapshots.go [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+			}).Error("failed to process region")
+			return resultMap, err
+		}
+		byteArr, err := json.Marshal(page)
+		if err != nil {
+			utilities.GetLogger().WithFields(log.Fields{
+				"tableName": "aws_rds_snapshots",
+				"account":   accountId,
+				"region":    *region.RegionName,
+				"task":      "DescribeDBSnapshots",
```
Comment (alende-uptycs) [inline extension/aws/rds/aws_rds_snapshots.go:172]: DescribeDBClusterSnapshots

File: extension/aws/rds/aws_rds_snapshots.go [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+		// table.TextColumn("snapshot_create_time_loc_tx"),
+		// table.IntegerColumn("snapshot_create_time_loc_tx_index"),
+		// table.TextColumn("snapshot_create_time_loc_tx_isstd"),
+		// table.TextColumn("snapshot_create_time_loc_tx_isutc"),
+		// table.BigIntColumn("snapshot_create_time_loc_tx_when"),
+		// table.TextColumn("snapshot_create_time_loc_zone"),
+		// table.TextColumn("snapshot_create_time_loc_zone_is_dst"),
+		// table.TextColumn("snapshot_create_time_loc_zone_name"),
+		// table.IntegerColumn("snapshot_create_time_loc_zone_offset"),
+		// table.BigIntColumn("snapshot_create_time_wall"),
```
Comment (alende-uptycs) [inline extension/aws/rds/aws_rds_snapshots.go:89]: Enable `SnapshotCreateTime`

File: generated/upt_cloud_rds_db_snapshots_inventory.txt [hunk 4]
```diff
@@ -0,0 +1,341 @@
+[
```
Comment (alende-uptycs) [inline generated/upt_cloud_rds_db_snapshots_inventory.txt:1]: Let's delete this file.

**Comment by alende-uptycs** [comment]
@dheeraj-uptycs  Update `Dockerfile` and `extension/aws/tables.md`

**Review by alende-uptycs — APPROVED** [review]

File: extension/plugins.go [hunk 5]
```diff
@@ -158,11 +158,13 @@ func RegisterPlugins(server *osquery.ExtensionManagerServer) {
 	server.RegisterPlugin(table.NewPlugin("aws_codedeploy_application", codedeploy.ListApplicationsColumns(), codedeploy.ListApplicationsGenerate))
 	// AWS CODECOMMIT
 	server.RegisterPlugin(table.NewPlugin("aws_codecommit_repository", codecommit.ListRepositoriesColumns(), codecommit.ListRepositoriesGenerate))
-	//AWS RDS
+	// AWS RDS
+	server.RegisterPlugin(table.NewPlugin("aws_rds_snapshot", rds.ListSnapshotsColumns(), rds.DescribeSnapshotsGenerate))
+	// AWS RDS
 	server.RegisterPlugin(table.NewPlugin("aws_rds_instance", rds.ListInstanceColumns(), rds.DescribeDBInstances))
+	//AWS RDS
```
Comment (alende-uptycs) [inline extension/plugins.go:165]: Let's remove these unwanted comments here

**Review by alende-uptycs — APPROVED** [review]

**Pull request merged**